source/sass/wagtail/blocks.scss

```diff
@@ -1,11 +1,15 @@
-.wide {
+.full-width {
```

Comment:
renaming wide to full-width, as this is the class that allows us to have the end to end content

I have also made this change in the image-block PR#6961, but since I was not sure which branch would get merged in first, I added it to this one as well